modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/ToolsEndpoint.java

```diff
@@ -284,8 +284,6 @@ private Boolean elementHasPreviewFlavor(MediaPackageElement element) {
                   @RestResponse(description = "Media package not found", responseCode = SC_NOT_FOUND) })
   public Response getVideoEditor(@PathParam("mediapackageid") final String mediaPackageId)
           throws IndexServiceException, NotFoundException {
-    if (!isEditorAvailable(mediaPackageId))
```

Comment:
This checks whether event.source is "ARCHIVE". While being processed, event.source is "WORKFLOW" though.

I don't think that it hurts to return editor.json while the event is being processed: This is a read-only request and the user already has opened the video editor anyway. Why shouldn't we allow that while an event is being processed?